### PR TITLE
avatar component

### PR DESCRIPTION
**Pull request by Shinde-Sandesh**
image

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
Looks good. You can merge.

File: Components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
+.VariantContainer{
+      margin-top: 25px;
+      border: 3px solid var(--primary-color);
+      width: 900px;
+      height: 200px;
+      display: flex;
+      flex-direction: row;
```
Comment (theVivek-kumar) [inline Components/avatar/avatar.css:7]: use rem and flex by default use raw so don't need to define raw

File: Components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      flex-wrap: wrap;
+      gap: 25px;
```
Comment (theVivek-kumar) [inline Components/avatar/avatar.css:9]: use rem here as well

File: Components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    height: 100px;
+    border-radius: 50%;
+  }
+  
+  .large{
+      width: 140px;
+      height: 140px;
+  }
+  
+  .medium{
```
Comment (theVivek-kumar) [inline Components/avatar/avatar.css:27]: use rem

File: Components/avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                      <img class="avatar small" src="Profile_avatar_placeholder_large.png" alt="">
+                      <img class="avatar medium" src="Profile_avatar_placeholder_large.png" alt="">
+                      <img class="avatar large" src="Profile_avatar_placeholder_large.png" alt="">
+                  </div>
+      
+                  <h3>Square Avatar</h3>
+                  <div class="VariantContainer">
+                      <img class="avatar-square square-small" src="Profile_avatar_placeholder_large.png" alt="">
+                      <img class="avatar-square square-medium" src="Profile_avatar_placeholder_large.png" alt="">
+                      <img class="avatar-square square-large" src="Profile_avatar_placeholder_large.png" alt="">
```
Comment (theVivek-kumar) [inline Components/avatar/avatar.html:42]: naming are good

File: Components/avatar/avatar.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                  </div>
+
+                  <h3>Oval Avatar</h3>
```
Comment (theVivek-kumar) [inline Components/avatar/avatar.html:45]: put image in a image folder

**Review by theVivek-kumar — CHANGES_REQUESTED** [review]
use em or rem in px rest can be merged

**Pull request merged**